lava_test_plans/projects/lt-qcom/fastboot.jinja2

```diff
@@ -21,7 +21,7 @@ protocols:
     name: lxc-target
     template: debian
     distribution: debian
-    release: stretch
+    release: buster
```

Comment:
why buster and not bullseye ?

Reply:
Even better would be to move away from lxc and start using docker.

Reply:
Let's change that to bullseye, agree.
Regarding lxc -> docker, I'd defer this question for now. Maybe @superna9999 can take a look.